src/robusta/core/sinks/slack/slack_sink.py

```diff
@@ -42,7 +42,7 @@ def handle_notification_grouping(self, finding: Finding, platform_enabled: bool)
             investigate_uri = self.get_timeline_uri(self.account_id, self.cluster_name)
             finding_data = finding.attribute_map
             # The top level entity name (the owner of the pod etc)
-            finding_data["workload"] = finding.service.name if finding.service else None
+            finding_data["workload"] = finding.service.name if finding.service else finding.backup_workload_name
```

Comment:
why are we doing it only for Slack?
it's probably useful on other places as well

Reply:
i saw that `workload` only exists for slack, we can do this for the robusta sink if you think too